tests/ut-graph.cc

```diff
@@ -773,14 +773,14 @@ TEST(graphapi, complete_extract_layer_gpu)
         + str_min_vals + ",\"max_vals\":" + str_max_vals
         + "},\"output\":{}},\"data\":[\"" + csvts_predict + "\"]}";
   joutstr = japi.jrender(japi.service_predict(jpredictstr));
-  std::cout << "joutstr=" << joutstr << std::endl;
+  // std::cout << "joutstr=" << joutstr << std::endl;
```

Comment:
This may be useful to debug the test if it fails during CI

Reply:
ok